docs/src/pages/system/styled/styled.md

```diff
@@ -93,3 +93,56 @@ const styled = createStyled({ defaultTheme });
 
 export default styled;
 ```
+
+## Difference with the `sx` prop
+
+We want the `styled` function to behave similarly to the `styled` function of other technologies, like Emotion or Styled Components. 
```

Comment:
```suggestion
The `styled` function is an extension of the `styled` utility provided by the underlying styled engine used, emotion or styled-components.
It is guaranteed that it will produce the same output as the `styled` function coming from the styled engine for the same input.
```